src/Pages/Details/Details.js

```diff
@@ -1,13 +1,39 @@
-import React from "react";
-import "./Details.scss"
-
-class Details extends React.Component{
-  render(){
-    return(
-    <div>
-    </div>
+import React, { Component } from "react";
+import InfoAndCartput from "./Components/InfoAndCartput";
+import ItemDescription from "./Components/ItemDescription";
+import DetailsInfo from "./Components/DetailsInfo";
+import ItemInquire from "./Components/ItemInquire";
+
+import "./Details.scss";
+
+const itemData = {
+  id: 1,
+  name: "[프로젝트]",
+  shortDescription: "DOG [어글어글] 우유껌 5종",
+  price: 5200,
+  sellUnit: "1개",
+  deliveryType: "샛별배송/택배배송",
+  originCountry: "국산",
+  packageType: "냉장/종이포장",
+  allergyInfo: [
+    "",
+    "본 제품은 돼지고기를 사용한 제품과 같은 제조시설에서 제조",
+  ],
+  shelfLif: "수령일 포함 4일 이상 남은 상품을 보내드립니다.",
+};
+
+class Details extends Component {
+  render() {
+    return (
+      <section className="details">
+        <div className="main-width">
+          <InfoAndCartput itemData={itemData}></InfoAndCartput>
+          <ItemDescription></ItemDescription>
+          <ItemInquire></ItemInquire>
```

Comment:
self-closing tag로 수정해주세요! JSX의 기본 특징입니다.
```suggestion
          <InfoAndCartput itemData={itemData} />
          <ItemDescription />
          <ItemInquire />
```

Reply:
넵